common/tags/src/test/java/org/apache/spark/tags/SQLTestGroupOne.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.tags;
+
+import org.scalatest.TagAnnotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@TagAnnotation
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
+public @interface SQLTestGroupOne { }
```

Comment:
May I ask why we choose this name, `SQLTestGroupOne`?

So, this PR proposes to have the following group name?
- (None) Default
- ExtendedSQLTest
- SQLTestGroupOne

Reply:
This is just because I didn't come up with a more suitable name, perhaps we could call it `SlowSQLTest`. Currently, most of the suites in this group contain test cases that run for more than 3 seconds (further screening is needed and some not slow ones need to be removed)

Reply:
Do you think about splitting by module which is more intuitive?

Reply:
According to this PR, all test suites under `package org.apache.spark.sql.streaming` seems to be in `SQLTestGroupOne`, doesn't it?

Reply:
> According to this PR, all test suites under `package org.apache.spark.sql.streaming` seems to be in `SQLTestGroupOne`, doesn't it?

No, just slower part with `SlowSQLTest `, for example, `ContinuousQueuedDataReaderSuite`, `ContinuousEpochBacklogSuite`, `LongOffsetSuite`, etc. are not with `SlowSQLTest `.

Reply:
> Do you think about splitting by module which is more intuitive?

I think it would be more intuitive. 
The advantage is that the grouping way is relatively flexible, which is conducive to further  group splitting in the future. The disadvantage is that multiple tags need to be added according to module and and the number of tags may continue to increase.